privilege-management.md

```diff
@@ -271,6 +271,10 @@ mysql> SELECT * FROM INFORMATION_SCHEMA.USER_PRIVILEGES WHERE grantee = "'root'@
 
 Requires the `SUPER` or `BACKUP_ADMIN` privilege.
 
+### CANCEL IMPORT JOB
+
+Requires the `SUPER` privilege to cancel jobs owned by other users. Otherwise, only jobs created by the current user can be canceled.
```

Comment:
```suggestion
Requires the `SUPER` privilege to cancel jobs created by other users. Otherwise, only jobs created by the current user can be canceled.
```